charts/loki-simple-scalable/values.yaml

```diff
@@ -59,68 +59,53 @@ loki:
   existingSecretForConfig: ""
   # -- Config file contents for Loki
   # @default -- See values.yaml
-  config: |
+  config:
```

Comment:
this should be a string

Reply:
this one won't fix due to line 63 auth_enabled: false is not string.

Reply:
we are moving in the direction of string config fields for all the charts. I don't understand what you mean about line 63, and us not being able to include `auth_enabed: false` in the config string?

without this fix this PR cannot be merged.